docs/04-resources/06-guides/powpeg/_category_.yml

```diff
@@ -0,0 +1,6 @@
+label: PowPeg App
+position: 2
+link:
+  type: generated-index
+  description: The PowPeg app fka 2 way peg app converts BTC to RBTC and vice versa. It is secured by the powpeg protocol, which is a unique PowPeg system that secures the locked bitcoins with the same Bitcoin hashrate that establishes consensus.
```

Comment:
application name should be: "PowPeg App"

Reply:
Updated. Thanks